client/quick-links/index.js

```diff
@@ -0,0 +1,190 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Card, CardBody, CardHeader, Icon } from '@wordpress/components';
+import Gridicon from 'gridicons';
+import { partial } from 'lodash';
+
+/**
+ * WooCommerce dependencies
+ */
+import { getSetting } from '@woocommerce/wc-admin-settings';
+import { List } from '@woocommerce/components';
+
+/**
+ * Internal dependencies
+ */
+import { recordEvent } from 'lib/tracks';
+import './style.scss';
+
+function getItems( props ) {
+	return [
+		{
+			title: __( 'Market my store', 'woocommerce-admin' ),
+			type: 'wc-admin',
+			path: 'marketing',
+			iconName: 'megaphone',
+			listItemTag: 'marketing',
+		},
+		{
+			title: __( 'Add products', 'woocommerce-admin' ),
+			type: 'wp-admin',
+			path: 'post-new.php?post_type=product',
+			iconName: 'gridicons-product',
+			listItemTag: 'add-products',
+		},
+		{
+			title: __( 'Personalize my store', 'woocommerce-admin' ),
+			type: 'wp-admin',
+			path: 'customize.php',
+			iconName: 'admin-customizer',
+			listItemTag: 'personalize-store',
+		},
+		{
+			title: __( 'Shipping settings', 'woocommerce-admin' ),
+			type: 'wc-settings',
+			tab: 'shipping',
+			iconName: 'gridicons-shipping',
+			listItemTag: 'shipping-settings',
+		},
+		{
+			title: __( 'Tax settings', 'woocommerce-admin' ),
+			type: 'wc-settings',
+			tab: 'tax',
+			iconName: 'gridicons-institution',
+			listItemTag: 'tax-settings',
+		},
+		{
+			title: __( 'Payment settings', 'woocommerce-admin' ),
+			type: 'wc-settings',
+			tab: 'checkout',
+			iconName: 'gridicons-credit-card',
+			listItemTag: 'payment-settings',
+		},
+		{
+			title: __( 'Edit store details', 'woocommerce-admin' ),
+			type: 'wc-settings',
+			tab: 'general',
+			iconName: 'store',
+			listItemTag: 'edit-store-details',
+		},
+		{
+			title: __( 'Get support', 'woocommerce-admin' ),
+			type: 'external',
+			href: 'https://woocommerce.com/my-account/create-a-ticket/',
+			iconName: 'sos',
+			listItemTag: 'support',
+		},
+		{
+			title: __( 'View my store', 'woocommerce-admin' ),
+			type: 'external',
+			href: props.getSetting( 'siteUrl' ),
+			iconName: 'external',
+			listItemTag: 'view-store',
+		},
+	];
+}
+
+function onItemClick( props, event ) {
```

Comment:
How about `handleOnItemClick` since `onItemClick` is already a prop?

Reply:
Will update.